

class ManageError(object):
    def __init__(self, max_iteration: int = 0):
        self.max_iteration = max_iteration
